apps/classroomio-com/src/routes/customer-education/+page.svelte

```diff
@@ -0,0 +1,200 @@
+<script>
+  import PageSignupCTA from '$lib/PageSignupCTA/index.svelte';
+  import PageHeader from '$lib/PageHeader/PageHeader.svelte';
+
+  const benefitDetails = [
+    {
+      img: 'personalised.svg',
+      title: 'Personalized Learning',
+      Description: 'Offer branded landing pages and multilingual courses for diverse customers.'
```

Comment:
for keys in objects please use camel case. Change `Description` to `description`